pom.xml

```diff
@@ -72,7 +72,7 @@
     <!--Version of grpc to be shaded -->
     <shaded.grpc.version>1.48.1</shaded.grpc.version>
     <!--Version of Netty to be shaded -->
-    <shaded.netty.version>4.1.77.Final</shaded.netty.version>
+    <shaded.netty.version>4.1.86.Final</shaded.netty.version>
```

Comment:
@MarcusWalzCollibra , thanks for working on this!   Let's also bump grpc version accordingly, see
https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Reply:
@szetszwo just bumped grcp! Thank you